regparser/tree/xml_parser/appendices.py

```diff
@@ -304,13 +314,22 @@ def is_subhead(tag, text):
             return self.m_stack.m_stack[0][0][1]
 
 
+_first_paren_markers = [re.compile(ur'[\)\.|,|;|-|—]\s*(\(' + lvl[0] + '\))')
```

Comment:
I would add examples (in comments) above this line, so that it's easy to know what this is trying to parse.